### DISCUSSION TITLE
blog/tension-between-internal-and-external-perspectives-in-political-learning/

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#55] [Announcements]
# blog/tension-between-internal-and-external-perspectives-in-political-learning/

这篇文章探讨了政治学习中的内外视角张力问题。作者认为，政治学习本质上应从内在视角出发，强调实践成员的承诺与参与，但过度依赖内在视角可能导致反思不足；引入外部视角虽能加深反思，但可能削弱实践承诺。文章旨在揭示如何平衡理论反思与实践参与，避免理论化过度或实践脱离的风险，以确保政治学习的有效性与忠诚度。

https://pathos.page/blog/tension-between-internal-and-external-perspectives-in-political-learning/

**Comment by savouer1** [comment]
这个问题，我觉得不是内外部视角的问题；而是被学习对象本身界定，和内外部的界定问题。
被学习的对象是不是阐述明确，且能够自洽；
被学习对象是不是稳定不变的；如果一直在变，那如何解决学习相对于对象的滞后性；
没有采用系统性学习和讨论，是否因为前两个问题都未能确定；
接上一个问题，如果不能系统性学习和讨论，而只能刷题库，且没有对刷题效果的考评，那如何保证大家能够用心的刷题？学生刷题是一种功利行为，所以尽管刷题的形式相比于系统学习来说效果较差，但还是有一定效果；但如果将刷题作为一种信仰，如何保证刷题的积极性？
那么到底应该先解决学习的积极性问题，还是先解决被学习对象的界定问题？学了一辈子，你问他，“你学的到底是什么”？有几个人能说清楚。

你说的学术问题，高校应该有一个具有决策权的学术委员会，但实际上这只是个装饰。

**Reply by Pathsis** [reply]
不清楚您这里所说的“学习”和我文章中所说的政治学习是不是一回事呀😄

**Reply by savouer1** [reply]
这个命题太大，能力驾驭不了，不讨论了。
